### ISSUE TITLE
Interactive highlight placement is odd

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/greenhouse-effect#239]
**Test device**
MacBook Air (m1 chip)

**Operating System**
macOS 13.0.1

**Browser**
safari 16.1

**Problem description**
For https://github.com/phetsims/qa/issues/871, this is very minor:
On the Layer Model screen, the interactive highlight for the Temperature checkbox is too large (or off center?). I can see that it is because the checkbox that was used in the same as that for the Layers, but in this case the mini thermometer was hidden behind the temperature reading. 

**Visuals**
Here is the Temperature checkbox:
<img width="338" alt="Screenshot 2023-01-06 at 10 12 46 AM" src="https://user-images.githubusercontent.com/87318828/211040520-d5bac6d8-7bb6-4384-b15d-889bfa77913a.png">

Here is the temperature checkbox for a layer:
<img width="283" alt="Screenshot 2023-01-06 at 10 09 35 AM" src="https://user-images.githubusercontent.com/87318828/211040165-67306088-3a31-424b-bb2d-24c9606a58ae.png">

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Greenhouse Effect‬
URL: https://phet-dev.colorado.edu/html/greenhouse-effect/1.1.0-rc.1/phet/greenhouse-effect_all_phet.html
Version: 1.1.0-rc.1 2022-12-23 00:15:29 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.1 Safari/605.1.15
Language: en-US
Window: 1399x710
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by jessegreenberg** [comment]
Thanks @Nancy-Salpepi. This highlight is the same as the focus highlight so it could be adjusted with the `focusHighlight` setter of ParallelDOM.ts. You could adjust it to surround the number display when visible with something like this:
```patch
Index: js/common/view/LayerModelObservationWindow.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/common/view/LayerModelObservationWindow.ts b/js/common/view/LayerModelObservationWindow.ts
--- a/js/common/view/LayerModelObservationWindow.ts	(revision 223b6c0266b58feff41ae231831c209b0dea012f)
+++ b/js/common/view/LayerModelObservationWindow.ts	(date 1673021803465)
@@ -8,6 +8,7 @@
  */
 
 import DerivedProperty from '../../../../axon/js/DerivedProperty.js';
+import { Shape } from '../../../../kite/js/imports.js';
 import optionize, { EmptySelfOptions } from '../../../../phet-core/js/optionize.js';
 import { Color, LinearGradient, Node, Path } from '../../../../scenery/js/imports.js';
 import soundManager from '../../../../tambo/js/soundManager.js';
@@ -119,6 +120,19 @@
     } );
     this.controlsLayer.addChild( surfaceThermometer );
 
+    // bounds containing the number display and the checkbox (in the local coordinate frame of the checkbox)
+    model.surfaceThermometerVisibleProperty.link( visible => {
+      if ( visible ) {
+        const numberDisplayLocalBounds = this.showThermometerCheckbox.globalToLocalBounds( surfaceThermometer.temperatureReadout.globalBounds );
+        const checkboxLocalBounds = this.showThermometerCheckbox.localBounds;
+        this.showThermometerCheckbox.focusHighlight = Shape.bounds( checkboxLocalBounds.union( numberDisplayLocalBounds ) );
+      }
+      else {
+        this.showThermometerCheckbox.focusHighlight = null;
+      }
+    } );
+
+
     // sound generation
     soundManager.addSoundGenerator( new AtmosphericPhotonsSoundGenerator( model.photonCollection, {
 
Index: js/common/view/ThermometerAndReadout.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/common/view/ThermometerAndReadout.ts b/js/common/view/ThermometerAndReadout.ts
--- a/js/common/view/ThermometerAndReadout.ts	(revision 223b6c0266b58feff41ae231831c209b0dea012f)
+++ b/js/common/view/ThermometerAndReadout.ts	(date 1673021231896)
@@ -178,7 +178,7 @@
       );
 
       // Create the temperature readout.
-      const temperatureReadout = new NumberDisplay( temperatureValueProperty, new Range( 0, 999 ), {
+      this.temperatureReadout = new NumberDisplay( temperatureValueProperty, new Range( 0, 999 ), {
         centerTop: thermometerNode.centerBottom.plusXY( 0, THERMOMETER_TO_READOUT_DISTANCE ),
         backgroundStroke: Color.BLACK,
         decimalPlaces: DECIMAL_PLACES_IN_READOUT,
@@ -195,7 +195,7 @@
           { units: unitsStringProperty }
         )
       } );
-      this.addChild( temperatureReadout );
+      this.addChild( this.temperatureReadout );
     }
 
     // mutate with layout options after the Node has been assembled

```

Or you could make the small checkbox icon invisible when the thermometer is displayed and the highlight should just surround just the checkbox.

Let me know if you would like to discuss.

**Comment by Nancy-Salpepi** [comment]
Also when the locale is Hausa, you can really see the thermometer icon peaking out.
<img width="173" alt="Screenshot 2023-01-09 at 9 30 37 AM" src="https://user-images.githubusercontent.com/87318828/211332047-875ca756-c923-4b9f-b003-13b925cc8b8d.png">

**Comment by jbphet** [comment]
Good eye @Nancy-Salpepi.  The surface thermometer was being positioned incorrectly, such that it overlapped the little thermometer icon.  This was due to some changes in the atmosphere layer node code, and I'd missed the unintended consequence.  A fix has now been committed to both the 1.1 and master branches.
